recipe/build.sh

```diff
@@ -8,6 +8,8 @@ unset SUBDIR
         --disable-doc \
         --enable-shared \
         --enable-static \
+        --enable-nonfree \
```

Comment:
Yeah, that clashes with `--enable-gpl`, which is used below for `x264` support, which is in turn needed to support things like `*.mp4` files.

Are you sure we need this flag @cailiang9? `openssl` is open source software so I don't think it should block our ability to link with GPL software as well. Could you please try dropping this flag to see if it works?